src/Point.java

```diff
@@ -1,6 +1,14 @@
 package traingame;
 
 public record Point(int q, int r) {
+    public int getQ() {
+        return q;
+    }
+
+    public int getR() {
+        return r;
+    }
+
```

Comment:
Because Point is a record, accessor functions are already provided. Iirc they should be callable by point.q() and point.r()

Reply:
Done.